packages/api/src/controllers/stream.js

```diff
@@ -117,6 +118,10 @@ app.get("/", authMiddleware({ admin: true }), async (req, res) => {
   if (active && active !== "false") {
     query.push(sql`stream.data->>'isActive' = 'true'`);
   }
+  if (nonLivepeerOnly && nonLivepeerOnly !== "false") {
+    // TODO make this work!
+    // query.push(sql`stream.data->>'user.email' NOT LIKE '%livepeer%'`);
```

Comment:
@darkdarkdragon could you give me a hand with this query? My SQL knowledge is a bit limited.

Reply:
That means "return only objects in which 'email' field does not contain 'livepeer' in it"

Reply:
sorry, I'm wrong, it should be written other way. one sec.

Reply:
I think i solved it! [https://github.com/livepeer/livepeer-com/pull/251/files#diff-3075aa9b906db2f795522df0e1996840d19743fddd8c8c5d41cce6393efac594R122](https://github.com/livepeer/livepeer-com/pull/251/files#diff-3075aa9b906db2f795522df0e1996840d19743fddd8c8c5d41cce6393efac594R122)